### PR TITLE
Added s.bus2 end bytes for (read only) use on s.bus2 port

### PR DESCRIPTION
**Pull request by Troynica**

### CODE REVIEW

File: library.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "authors":
+  [
+    {
+      "name": "Dennis Marinus",
+      "email": "dennis@marinus.nu",
+      "maintainer": true
+    }
+  ],
+  "dependencies":
```
Comment (zendes) [inline library.json]: this looks like it's missing a `{`, and a matching closing one after platforms

**Review by zendes — CHANGES_REQUESTED** [review]
library.json isn't valid json

File: library.json (hunk 1 above)
Comment (Troynica) [reply library.json]: Yeah, aside from that, I should not have added my library.json in the first place. Removed it from the request...

Comment (Troynica) [reply library.json]: Created a new library.json after all.

**Comment by Troynica** [comment]
Corrected SBUS.cpp

File: library.json (hunk 1 above)
Comment (Troynica) [reply library.json]: Keyword "dependencies" was missing a value. Removed in later version.

**Pull request closed without merging**